submitting project info

brmazzoni opened the pull request.

brmazzoni committed:
submitting project info

OliverMensahDev reviewed (changes requested):
Awesome work, @brmazzoni 

Notice you used terraform in creating your clusters. 
Kindly include in your write-up some screenshots of using the cluster in your project.

brmazzoni commented:
> Awesome work, @brmazzoni
> 
> Notice you used terraform in creating your clusters. Kindly include in your write-up some screenshots of using the cluster in your project.

Hi @OliverMensahDev , thanks for your review.
I updated the README with some screenshots, fixed an error, added a warning about security.
https://github.com/brmazzoni/curly-rotary-phone/blob/main/README.MD
